guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,352 @@
+- Generating a payjoin-enabled payment link is automatic once it is setup
+- If the user chooses to add an existing hot wallet, BTCPay does not provide an option to enable payjoin on the same page like it does during new hot wallet creation
+- If a user opts for a watch-only (cold) wallet during setup (new or existing), they will never encounter payjoin (because a wallet is required in order to sign the payjoin transactions)
+- Only one bitcoin wallet can be connected to the store (except through [BTCPay Vault](https://docs.btcpayserver.org/HardwareWalletIntegration/)) so using a cold wallet precludes setting up a hot wallet (and Payjoin)
+- BTCPay does not prompt or warn the user if there are no funds in the hot wallet to perform a payjoin, resulting in failed payments
+- There is no dedicated space to manage and monitor Payjoin
+
+Read the full analysis on [receiving payjoin payments on BTCPay](https://docs.google.com/document/d/1_de2pkMREGpZQwOefQdH-MZNSUPazObgRK1cruiWBmo/edit?usp=sharing) here.
+
+
+#### Proposed Receiver Flow
+
+Here we will devise user flows for a POS system that can always be online. This excludes mobile wallets.
+
+A receiver should be able to set up payjoin during onboarding on the platform (app or POS system) or any time after that. The following is a standalone payjoin setup flow.
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin-case-study/payjoin-receiver-flow@2x.png"
+   retina = "assets/images/guide/case-studies/payjoin-case-study/payjoin-receiver-flow@2x.png"
+   alt-text = "payjoin receiver flow"
+   width = 800
+   height = 458
+%}
+
+A detailed process flow integrated with onboarding is available in this [Figjam file](https://www.figma.com/file/NzMvwyzP7x5jfGmwNUKRov/PayJoin-Process-Flows?node-id=0%3A1&t=wSwewTOkQddWhagl-1).
+
+##### Requesting payjoin
+
+Once the payjoin setup is functional, all payment requests (BIP-21 payment links or encoded as QRs) should contain the necessary information (including endpoint address) to perform the payjoin transaction. No separate user action should be required.
+
+
+#####  Payjoin Status/Settings
+
+A dedicated section in the settings page might be a good idea to provide dedicated space in the UI to manage Payjoin settings and monitor its status. This would also provide UI space for users using a cold wallet to discover and set up payjoin. This section could contain:
+
+- Payjoin setup workflow
+- Additional payjoin features like third party endpoints, address substitution, cold-wallet backfill
+- Monitor payjoin readiness: hot wallet status, pj-endpoint status etc.
+
+
+## Insights
+
+We learnt many things over the course of this case study, and the user flows we have designed already use many of these insights.
+
+- Payjoins are a great way to consolidate UTXOs so they pay lower fees in the long run – this can be especially useful to exchanges, trading desks who need this the most and possess technical ability to implement a robust setup
```

Comment:
```suggestion
- Payjoins are a great way to consolidate UTXOs so participants pay lower fees in the long run. This can be especially useful for trading desks at exchanges who need this the most, and possess the technical ability to implement a robust setup
```